app.js

```diff
@@ -1,59 +1,135 @@
+const clients = [];
 
 const addBtn = document.getElementById('add');
+const resetBtn = document.getElementById('reset');
 const accntName = document.getElementById('accntName');
 const accntNo = document.getElementById('accntNo');
+const typeOfAccnt = document.querySelector('input[name="accountType"]:checked');
```

Comment:
`const form = document.forms.bank;`
`const radios = form.elements.accountType;`

_change line 7 and use above codes, this will ensure that correct value of radio is selected_